src/plone/app/mosaic/profiles/default/registry.xml

```diff
@@ -233,6 +234,17 @@
     <value key="weight">220</value>
   </records>
 
+  <records interface="plone.app.mosaic.interfaces.IFormat"
```

Comment:
After adding records an upgrade step loading the new record into existing installations is necessary.

Reply:
added upgrade step